src/toil/fileStores/abstractFileStore.py

```diff
@@ -213,7 +213,7 @@ def getLocalTempDir(self) -> str:
         """
         return os.path.abspath(tempfile.mkdtemp(dir=self.localTempDir))
 
-    def getLocalTempFile(self) -> str:
+    def getLocalTempFile(self, suffix=None, prefix=None) -> str:
         """
         Get a new local temporary file that will persist for the duration of the job.
 
```

Comment:
We could use docstring descriptions of the new parameters here and in `getLocalTempFileName`.